src/main/java/eggventory/commands/DeleteCommand.java

```diff
@@ -24,11 +24,11 @@ public void execute(StockType list, Ui ui, Storage storage) throws IndexOutOfBou
         try {
             Stock item = list.getStock(itemIndex);
             list.deleteStock(itemIndex);
-            ui.print("Okay! I've deleted this task:\n" + item.toString());
+            ui.print("Okay! I've deleted this stock:\n" + item.toString());
```

Comment:
I feel we should immediately execute a save after doing a delete

Reply:
i think we decided to leave saving only at the end for now